.changeset/cuddly-shoes-camp.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
We can delete this changeset. Since only the internals of the components (in this case just the icon name) has changed, the consumers don't need to change anything in their code. The other changeset will patch bump dependent packages using icons automatically.

Reply:
ah ok, shoot. I accidentally deleted the others before I saw your comment. Will fix.